assemble/conf/templates/accumulo-env.sh

```diff
@@ -25,6 +25,21 @@
 # Variables that must be set
 ############################
 
+## Accumulo logs directory. Referenced by logger config.
+export ACCUMULO_LOG_DIR="${ACCUMULO_LOG_DIR:-$ACCUMULO_HOME/logs}"
+## Hadoop installation
+export HADOOP_PREFIX="${HADOOP_PREFIX:-/path/to/hadoop}"
+## Hadoop configuration
+export HADOOP_CONF_DIR="${HADOOP_CONF_DIR:-${HADOOP_PREFIX}/etc/hadoop}"
+## Zookeeper installation
+export ZOOKEEPER_HOME="${ZOOKEEPER_HOME:-/path/to/zookeeper}"
+## See HADOOP-7154 and ACCUMULO-847
+export MALLOC_ARENA_MAX=${MALLOC_ARENA_MAX:-1}
```

Comment:
I think leaving it in the environment file was fine. That *was* where the reasonable default is. I think putting it in the `accumulo` script is inappropriate, as it's not an Accumulo thing at all, and that basically goes back to what we had before, with config everywhere and hard to get a handle on "what is the default, and where is the default defined, and what overrides what?".

It's a JVM thing in some environments. Setting it in the default environment file makes sense to me.